pkg/checks/update.go

```diff
@@ -64,6 +70,66 @@ func (o CheckUpdateOptions) CheckUpdates(files []string) error {
 
 	return checkErrors.WrapErrors()
 }
+
+// validates update configuration
+func validateUpdateConfig(files []string, checkErrors *lint.EvalRuleErrors) {
+	for _, file := range files {
+		// first need to read raw bytes as unmarshalling a struct without a pointer means update will never be nil
+		if !strings.HasSuffix(file, ".yaml") {
+			file += ".yaml"
+		}
```

Comment:
Just curious why we would need to account for this case (or not be sure about the input state)

Reply:
mainly ease of use, so when running the check command you don't have to keep adding `.yaml`.  With the developer workflow I tend to copy the melange package name and then paste into my terminal to run the command.  Just means folks can specify package name or file name.